pkg/api/apis/operators/v1alpha2/operatorgroup_types.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type OperatorGroupSpec struct {
+	Selector       metav1.LabelSelector  `json:"selector,omitempty"`
+	ServiceAccount corev1.ServiceAccount `json:"serviceAccount,omitempty"`
+}
+
+type OperatorGroupStatus struct {
+	Namespaces  []*corev1.Namespace `json:"namespaces"`
```

Comment:
I'm still a little concerned that this could get too long. `[]string` seems slightly safer. Perhaps the real answer is to wait for CRDs to support arbitrary subresources and implement a `/namespaces` subresource (so that we can paginate)